react-app/src/library/services/application.ts

```diff
@@ -0,0 +1,98 @@
+import { JabraNativeCommands } from './vendor-implementations/jabra/jabra-native/jabra-native-commands';
+
+// TODO: This is just a shell for now to make things build
+const requestDesktopPromise = (cmd) => {
+  return (resolve, reject) => {
+    try {
+      let sCmd = JSON.stringify(cmd);
+      (window as any).cefQuery({
+        request: sCmd,
+        persistent: false,
+        onSuccess: response => {
+          try {
+            let obj = JSON.parse(response);
+            resolve(obj);
+          } catch (e) {
+            resolve({});
+          }
+        },
+        onFailure: response => {
+          reject(response);
+        }
+      })
+    } catch (e) {
+      console.error('Error requesting desktop promise', e);
+      reject();
```

Comment:
This was a line based off of what was in the ESI... The caller has a catch block in case of error that seems to have it's own error that it logs before calling the `disconnect` function of the file in question